ksm/ksm_day3/문자열 다루기 기본.js

```diff
@@ -0,0 +1,6 @@
+function solution(s) {
+    
+    return (s.length==4 || s.length==6) && /^[0-9]*$/.test(s);
+} 
```

Comment:
```suggestion
function solution(s) {
    const regex = /^[0-9]*$/;
    return (s.length === 4 || s.length === 6) && regex.test(s);
} 
```
정규 표현식은 다른 변수로 분리하면 더 가독성을 높일 것 같아요.
js에서 `==` 연산자는 거의 안쓰인다고 보시면 되요 대부분의 경우 `===`이 쓰이는 점 기억해주세요.
- 물론 ==도 쓰이는 경우도 있습니다 ㅎㅎ